### DISCUSSION TITLE
Module parse failed: Unexpected token (??:??)

### DISCUSSION THREAD

**Discussion by BranchScope** [themesberg/flowbite-react#744] [General]
**Describe the bug**
I got this error while compiling with this new import:
`import { Navbar } from 'flowbite-react';`
The fact is that you actually answered others issues saying that we are using an older version of EcmaScript, but you didn't show us how to change/update it. I can't find anything on Google. Can you help us?

**Screenshots**
![image](https://github.com/themesberg/flowbite-react/assets/39023184/573b0692-c7ef-4f29-8881-98a5e618063c)

**Project information:**

- Tailwind: 3.3.2
- Flowbite: 1.6.5
- Flowbite React: 0.4.4

**Comment by BranchScope** [comment]
**UPDATE**
I've just tried to switch to yarn in order to fix all the errors i had with npm, but here's the result:
![image](https://github.com/themesberg/flowbite-react/assets/39023184/c4e9b8b5-80eb-4b73-a702-43fd7bf4b5d0)

**package.json**
```json
{
  "name": "test",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@heroicons/react": "^2.0.18",
    "@material-tailwind/react": "^2.0.0",
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "flowbite": "^1.6.5",
    "flowbite-react": "^0.4.4",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-router-dom": "^6.11.1",
    "react-scripts": "^2.1.3",
    "web-vitals": "^2.1.4"
  },
  "scripts": {
    "start": "react-scripts --openssl-legacy-provider start", #line modified because i have node v20 now which is unsupported by something
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "tailwindcss": "^3.3.2"
  }
}
```

**Comment by mSulleza** [comment]
Any updates on this? I'm getting the same error.

**Reply by BranchScope** [reply]
nope.

**Reply by tulup-conner** [reply]
Please refer to the pinned discussion at https://github.com/themesberg/flowbite-react/discussions/561 which has almost the same exact title.

**Reply by tulup-conner** [reply]
I'm not really sure what you mean by help you change your version of ECMAScript. I can't do that for everyone because there isn't a universal fix. Most likely this is a niche configuration in something in your stack - that's on you to fix
